tools/cpplint.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.4
+#!/usr/bin/env python
```

Comment:
Not that I care much, but this wouldn't work on Arch Linux where `python` is python3.